master/master.cfg

```diff
@@ -386,6 +401,16 @@ def prioritizeBuilders(buildmaster, builders):
 
 c['prioritizeBuilders'] = prioritizeBuilders
 
+# STYLE BUILD SLAVES
+numstyleslaves = 1
+
+style_slaves = [
+    ZFSEC2BuildSlave(
+        name="Amazon-2015.09-x86_64-styleslave%s" % (str(i+1)),
+        ami="ami-469edd26"
```

Comment:
s/ami-469edd26/ami-f1d69891/ - I've just updated the Amazon AMI to resolve build failures due to them updating their kernel.